src/backend_model.cc

```diff
@@ -488,6 +600,16 @@ TritonModel::~TritonModel()
   }
 }
 
+void
+TritonModel::ClearHandles()
+{
+  batch_dlhandle_ = nullptr;
```

Comment:
Should call `CloseLibraryHandle()` to actually release the library handle. It is strange that we are just setting to `nullptr` in the `backend_manager.cc` as well, which is not correct based on my understanding. @Tabrizian @tanmayv25 ^^

Reply:
Thank you for clarifying. Updated to use CloseLibraryHandle().

Reply:
I guess maybe because we don't load/unload backend shared libraries anymore, we are relying on the application shutdown to close the backend shared libraries?